packages/apollo/src/utils/getOperationManifestFromProject.ts

```diff
@@ -27,54 +16,22 @@ export function getOperationManifestFromProject(
   const manifest = Object.values(
     project.mergedOperationsAndFragmentsForService
   ).map(operationAST => {
-    // While this could include dropping unused definitions, they are
-    // kept because the registered operations should mirror those in the
-    // client bundle minus any PII which lives within string literals.
-    const printed = printWithReducedWhitespace(
-      sortAST(hideCertainLiterals(operationAST))
-    );
+    const printed = defaultEngineReportingSignature(operationAST, "");
```

Comment:
I'm confused about this change. This seems to be moving operation registry normalization in a more aggressive/lossy direction instead of a more conservative direction. This adds removeAliases and changes hideCertainLiterals to the stronger hideLiterals.  (And dropUnusedDefinitions though I think they never show up due to the operation registry's construction.)

Reply:
Yeah, I believe this should be whatever `hashForOperationSignature` ends up being called.

Reply:
@glasser you're correct - I believe the right move here then is to make `defaultEngineReportingSignature` less aggressive as you're suggesting.

```javascript
return printWithReducedWhitespace(
  sortAST(hideCertainLiterals(dropUnusedDefinitions(ast, operationName)))
);
```